packages/page-experience/lib/helpers/parseFontfaceSrc.js

```diff
@@ -0,0 +1,37 @@
+const fontfaceSrcParser = require('css-font-face-src');
+
+/**
+ * Extracts the best fontface URL for preloading (woff2 > woff > ?).
```

Comment:
Smart here: I was naively thinking the result would always be `woff2` but this code correctly validates if there are other formats available to use.